controllers/shipwrightbuild_controller.go

```diff
@@ -233,6 +252,16 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		err = r.Client.Status().Update(ctx, b)
 		return RequeueWithError(err)
 	}
+
+	err = deleteObjectsIfPresent(ctx, r.Client, []client.Object{
+		&rbacv1.ClusterRoleBinding{ObjectMeta: metav1.ObjectMeta{Name: "shipwright-build-webhook"}},
+		&rbacv1.ClusterRole{ObjectMeta: metav1.ObjectMeta{Name: "shipwright-build-webhook"}},
+	})
+	if err != nil {
+		logger.Error(err, "deleting'shipwright-build-webhook' role and cluster role binding")
```

Comment:
nit - spelling and avoid use of quote characters. Logger tends to use structured format: 
```suggestion
		logger.Error(err, "deleting shipwright-build-webhook ClusterRole and ClusterRoleBinding")
```